### ISSUE TITLE
Indentation of code

### ISSUE THREAD

**Issue by brunowego** [virtyaluk/grunt-iife#2]
Hello again, the result is great but not indented, the linters show errors with the indentation.

``` js
;(function() {
'use strict';

/**
 * Chip
 */

angular
    .module('xx.chip', [
        'ngSanitize'
    ]);

    ...
}());
```

Try check with JSHint, ESLint, JSCS...

**Comment by virtyaluk** [comment]
Hey @brunowego ! Thanks again for interest to my project.

As you have mentioned the output code was wrapped into IIFE with a **broken indentation**. The reason for such behaviour is because _grunt-iife_ is a **post-processing task**. Which means, by first you write your code, after this you check it using favorite linter, and only after these you wrap it using _grunt-iife_ in conjunction with others post-processing tasks (uglify and so).

Once again, thanks for your feedback.
For now I close this one. If you have another questions you may ask these on here or by opening another issue.

Be seeing you :smile:

**Comment by brunowego** [comment]
I have small proposal for you @virtyaluk, is a small change, think about it. In my case I need the code clean, validations on the server. #3
